membership.py

```diff
@@ -51,16 +51,20 @@ def compare(dicts):
     return common_members
 
 
-def copy_qtgui_to_qtwidgets():
+def copy_qtgui_to_modules():
     """Copies the QtGui list of PySide/PyQt4 into QtWidgets"""
 
     pyside_filepath = PREFIX + '/PySide.json'
     pyqt4_filepath = PREFIX + '/PyQt4.json'
     pyside = read_json(pyside_filepath)
     pyqt4 = read_json(pyqt4_filepath)
 
+    # When Qt4 was moved to Qt5, they split QtGui into QtGui, QtWidgets, and
+    # QtPrintSupport.
     pyside['QtWidgets'] = pyside['QtGui']
     pyqt4['QtWidgets'] = pyqt4['QtGui']
+    pyside['QtPrintSupport'] = pyside['QtGui']
+    pyqt4['QtPrintSupport'] = pyqt4['QtGui']
 
     write_json(pyside, pyside_filepath)
     print('--> Copied QtGui to QtWidgets for ' + os.path.basename(
```

Comment:
We should change this print message to also mention `QtPrintSupport`.

Reply:
Heh, I guess I completely ignored the print statement.

Reply:
It now outputs:
```
C:\blur\dev\Qt_fork>membership.py --copy-qtgui
--> Wrote PySide.json
--> Copied QtGui to QtWidgets and QtPrintSupport for PySide.json
--> Wrote PyQt4.json
--> Copied QtGui to QtWidgets and QtPrintSupport for PyQt4.json
```